commune_app/templates/commune_app/logout.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
I think the logout should return to the main page 
IMHO we don't need this page